src/Daemon/BaseDaemon.cpp

```diff
@@ -504,24 +504,17 @@ void BaseDaemon::defineOptions(Poco::Util::OptionSet & new_options)
 
 void BaseDaemon::handleSignal(int signal_id)
 {
-    if (signal_id == SIGINT ||
+    if (!(signal_id == SIGINT ||
```

Comment:
Hmm, what are these changes for?

Reply:
https://s3.amazonaws.com/clickhouse-test-reports/PRs/80332/708a5aa8e3b739ffd9528b767a9e484537c04d1d//integration_tests_tsan_3_6/integration_run_test_system_logs_test_system_logs_py_0.log

Reply:
https://clickhouse-inc.slack.com/archives/C02N5V7EU57/p1747429246114119?thread_ts=1747333914.981179&cid=C02N5V7EU57